src/screens/About/TeamTable/TeamTable.tsx

```diff
@@ -1,25 +1,44 @@
 import React from 'react';
 
 import { TeamTableGrid, TeamMember, TeamMemberCell } from './TeamTable.style';
+import {
+  TeamMemberName,
+  TeamMemberTitle,
+} from '../HeadshotGrid/HeadshotGrid.style';
 
 const TeamTable = ({
   people,
+  isTeam,
+  isAdditionalAdvisors,
```

Comment:
If `<TeamMember>` doesn't actually use `isAdditionalAdvisors` (per my comments above), then I don't think we need it here either?